ts/features/pn/components/PnMessageDetails.tsx

```diff
@@ -67,21 +63,14 @@ export const PnMessageDetails = (props: Props) => {
   const currentFiscalCode = useIOSelector(profileFiscalCodeSelector);
   const frontendUrl = useIOSelector(PnConfigSelector).frontend_url;
 
-  const maybePayment = props.message.recipients.find(
-    _ => _.taxId === currentFiscalCode
-  )?.payment;
-
-  const noticeNumber = PaymentNoticeNumber.decode(maybePayment?.noticeCode);
-  const creditorTaxId = OrganizationFiscalCode.decode(
-    maybePayment?.creditorTaxId
+  const maybePayment = pipe(
```

Comment:
Per my understanding, "maybe[name]" is used when dealing with `Option` types. Since this is not the case, should we rename it to something else?

Reply:
what do you think about `payment`?

Reply:
Good enough for me

Reply:
changed https://github.com/pagopa/io-app/pull/4624/commits/c402ece1905e66705b52f03fb6bfe1c4d3c42549